pkg/adapter/awscodecommitsource/adapter.go

```diff
@@ -216,68 +216,49 @@ func (a *adapter) preparePullRequests() ([]*codecommit.PullRequest, error) {
 		RepositoryName: &a.arn.Resource,
 	}
 
-	for {
-		//Get pull request IDs
-		pullRequestsOutput, err := a.ccClient.ListPullRequests(&input)
-		if err != nil {
-			return pullRequests, fmt.Errorf("failed to list PRs: %w", err)
-		}
-
-		prIDs := []*string{}
-
-		prIDs = append(prIDs, pullRequestsOutput.PullRequestIds...)
-
-		for _, id := range prIDs {
-			pri := codecommit.GetPullRequestInput{PullRequestId: id}
-
-			prInfo, err := a.ccClient.GetPullRequest(&pri)
-			if err != nil {
-				return pullRequests, fmt.Errorf("failed to get PR info: %w", err)
-			}
+	//Get pull request IDs
```

Comment:
A convention more than a written rule, but it's idiomatic to have a leading space for comments that are not meant to be consumed by machines (https://github.com/golang/go/issues/37974#issuecomment-607510041).